otus/freebsd/src/sys/dev/athp/if_athp_pci_hif.c

```diff
@@ -1202,6 +1202,7 @@ ath10k_pci_hif_power_up(struct ath10k *ar)
 	ath10k_pci_ce_deinit(ar);
 
 err_sleep:
+	//ath10k_pci_sleep(ar_pci);
```

Comment:
Why this?

Reply:
I just wanted it to know that it was in the original driver, but it can be removed.

Reply:
removed comment.